src/store/cart.js

```diff
@@ -0,0 +1,32 @@
+const initialState = {
+  cartItems: 0
+}
+
+export default function reducer ( state=initialState, action) {
+  const { type, payload} = action;
+
+  switch(type) {
+    
+    case 'INCREMENT':
+      return {...state, cartItems: state.cartItems + 1}
+
+      case 'DECREMENT':
+      return { ...state, cartItems: state.cartItems - 1}
+
+      
+      default:
+        return state;
+  }
+}
+
+export const increment = () => {
+  return{
+    type: 'INCREMENT'
+  }
+}
+
+export const decrement = () => {
+  return{
+    type: 'DECREMENT'
+  }
+}
```

Comment:
You should be adding the selected product to the cart. Right now your cart acts more of a counter then a cart.